tests/testNRyield.cc

```diff
@@ -49,15 +55,22 @@ void testNRyield(G4double Emin, G4double Emax, const G4String& unit,
   if (!goodInput) ::exit(1);		// If anything failed, abort
 
   // Instantiate single instance of each of the named yield functions
-  const char* useNIEL[] = { "Lindhard", "LewinSmith", "Sarkis", "Impact" };
+  const char* useNIEL[] = {"Lindhard", "Empirical Lindhard", "LewinSmith", "Sarkis", "Impact" };
+  // const char* useNIEL[] = {"Empirical Lindhard"};
   const size_t nNIEL = sizeof(useNIEL)/sizeof(char*);
 
   // NOTE: Can't use G4CMPConfigManager to do this mapping, because it
   //       deletes the previous pointer when a new one is requested.
-  const G4VNIELPartition* NIELfunc[nNIEL] = {
-    new G4CMPLindhardNIEL, new G4CMPLewinSmithNIEL,
-    new G4CMPSarkisNIEL, new G4CMPImpactTunlNIEL };
+     
 
+const G4VNIELPartition* NIELfunc[nNIEL] = {
+    new G4CMPLindhardNIEL, new G4CMPEmpiricalLindhardNIEL(),
+    new G4CMPLewinSmithNIEL, new G4CMPSarkisNIEL,
+    new G4CMPImpactTunlNIEL };
+
+
+
+  // const G4VNIELPartition* NIELfunc[nNIEL] = {empiricalLindhard};
```

Comment:
Please remove the extra blank lines and the commented out code.  There should be exactly one blank line before the next comment.